modules/nixos/main.py

```diff
@@ -805,6 +533,12 @@ def run():
 
     # Write the configuration.nix file
     libcalamares.utils.host_env_process_output(["cp", "/dev/stdin", config], None, cfg)
+    # Format configuration file
+    libcalamares.utils.host_env_process_output(["nixfmt",  config], None)
```

Comment:
Have you applied my proposed module changes before building the iso? https://github.com/FriedrichAltheide/nixpkgs/commits/calamares-nixos-extensions-custom-installer/

Reply:
Is there a reason we can’t just ensure the templates we ship are formatted so that the resulting config will be without an explicit formatting step?

Reply:
It would be possible to format the default snippets in a way that the default configuration would look nicely. Yet, this PR intends to add an easy way to build custom installers based on the default calamares installer (e.g, some default settings like proxy, SSO, or CA settings). Hence, it would be nice to format the generated configuration.nix, ensuring that it is formatted nicely for new users.